pegasus/src/homepage.rb

```diff
@@ -27,27 +27,23 @@ def self.get_announcement_for_page(page)
   end
 
   def self.load_announcements
-    # Reloads JSON file with announcement data on each page load
-    # in non-production environments
-    unless (@@load_error || @@loaded) && rack_env?(:production)
-      unless File.file?(@@json_path)
-        @@load_error = true
-        return
-      end
-      begin
-        @@announcements_data = JSON.parse(
-          IO.read(@@json_path),
-          symbolize_names: true,
-          object_class: HashWithIndifferentAccess
-        )
-        unless validate_announcements_data(@@announcements_data)
-          @@load_error = true
-        end
-      rescue JSON::ParserError
+    unless File.file?(@@json_path)
+      @@load_error = true
+      return
+    end
+    begin
+      @@announcements_data = JSON.parse(
```

Comment:
is there any concern with this slowing down load times if we load every time?

Reply:
We have always heavily cached our signed-out homepage (and indeed all pegasus content), so I don't think this should be an issue.